xlsx_inventory.py

```diff
@@ -68,13 +85,13 @@ def create_config(filename=None, group_by_col=None, hostname_col=None, sheet=Non
         config.write(cf)
```

Comment:
On line 84 the variable `config_file` instead of the newly introduced `config_path` is used.
This is inconsistent, and leads to an FileNotFound Exception down the line, when the `--config` option is used and the Environment Variable `EXCEL_INVENTORY_CONFIG` is set.

Would you recommend restructuring the argparse and `create_config()` logic to accommodate for this, or would you recommend to alter just this line of Code for consistency?

Thank you.

Reply:
If I'm synced up on everything here, the use case this is supporting will be:

```
EXCEL_INVENTORY_CONFIG=foo.cfg ./xlsx_inventory.py --config --file example.xlsx --hostname-col A --group-by-col B --sheet Sheet1
```

I will push a change to make this work to write to `foo.cfg`.